pow/src/compute/v1.rs

```diff
@@ -55,10 +55,10 @@ impl ComputeV1 {
 			mode,
 		);
```

Comment:
Just looks slightly weird here. Prefer slightly to move the try here.
```suggestion
		)?;
```